main.js

```diff
@@ -14,20 +14,25 @@ const handleReceivedMessage = event => {
 };
 
 const checkSmartmatchProp = () => {
-	// Is this code running on a Smartmatch-compatible page?
-	const pageUrl = window.top.location && window.top.location.href;
-	if (!pageUrl) {
-		sendMonitoringEvent('Top window location info inaccessible');
-		return;
-	}
+	try {
+		// Is this code running on a Smartmatch-compatible page?
+		const pageUrl = window.top.location && window.top.location.href;
+		if (!pageUrl) {
+			sendMonitoringEvent('Top window location info inaccessible');
+			return;
+		}
 
-	const isSMpage = pageUrl.match(/ft.com\/content/);
+		const isSMpage = pageUrl.match(/ft.com\/content/);
 
-	if (isSMpage) {
-		const hasSMObjOnLoad = Boolean(window.top.smartmatchCreativeMatches);
-		const neg = hasSMObjOnLoad ? ' ' : ' NOT ';
-		sendMonitoringEvent(`SM obj was${neg}available when iframe loaded`);
+		if (isSMpage) {
+			const hasSMObjOnLoad = Boolean(window.top.smartmatchCreativeMatches);
```

Comment:
maybe can be simplified to double-not?
```suggestion
			const hasSMObjOnLoad = !!(window.top.smartmatchCreativeMatches);
```

Reply:
The origami eslint rules changed that automatically. I don't like it either